business_requirement/models/business.py

```diff
@@ -0,0 +1,277 @@
+            'confirmed': [('readonly', False)]}
+    )
+    level = fields.Integer(
+        compute='_get_level',
+        string='Level',
+        store=True
+    )
+    change_request = fields.Boolean(
+        string='Change Request?',
+        default=False,
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    project_id = fields.Many2one(
+        comodel_name='project.project',
+        string='Master Project',
+        ondelete='set null',
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    partner_id = fields.Many2one(
+        comodel_name='res.partner',
+        string='Customer',
+        store=True,
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    sub_br_count = fields.Integer(
+        string='Count',
+        compute='_sub_br_count'
+    )
+    priority = fields.Selection(
+        [('0', 'Low'), ('1', 'Normal'), ('2', 'High')],
+        'Priority',
+        required=True,
+        default='1'
+    )
+    requested_id = fields.Many2one(
+        'res.users',
+        string='Requested by',
+        required=True,
+        readonly=True,
+        default=lambda self: self.env.user,
+        states={
+            'draft': [('readonly', False)],
+            'confirmed': [('readonly', False)]}
+    )
+    confirmation_date = fields.Datetime(
+        string='Confirmation Date',
+        readonly=True
+    )
+    confirmed_id = fields.Many2one(
+        'res.users', string='Confirmed by',
+        readonly=True
+    )
+    reviewed_date = fields.Datetime(
+        string='Reviewed Date',
+        readonly=True
+    )
+    reviewed_id = fields.Many2one(
+        'res.users', string='Reviewed by',
+        readonly=True
+    )
+    approval_date = fields.Datetime(
+        string='Approval Date',
+        readonly=True
+    )
+    approved_id = fields.Many2one(
+        'res.users',
+        string='Approved by',
+        readonly=True
+    )
+    company_id = fields.Many2one(
+        'res.company', string='Company',
+        required=True, readonly=True, states={'draft': [('readonly', False)]},
+        default=_get_default_company,
+    )
+
+    @api.multi
+    @api.onchange('project_id')
+    def project_id_change(self):
+        if self.project_id and self.project_id.partner_id:
+            self.partner_id = self.project_id.partner_id.id
+
+    @api.model
+    def create(self, vals):
+        if vals.get('name', '/') == '/':
+            vals['name'] = self.env['ir.sequence'].get('business.requirement')
+        return super(BusinessRequirement, self).create(vals)
+
+    @api.multi
+    @api.depends('parent_id')
+    def _get_level(self):
+        def _compute_level(br):
+            return br.parent_id and br.parent_id.level + 1 or 1
+
+        for br in self:
+            level = _compute_level(br)
+            br.level = level
+
+    @api.multi
+    @api.depends('business_requirement_ids')
+    def _sub_br_count(self):
```

Comment:
Following the guideline, this should be name `_compute_...`
I believe this is being enforced on newer builds.